tests/save-test-results.sh

```diff
@@ -16,6 +16,7 @@
 # Saves test results to reporting server
 
 echo "rpcbind : $REPORTING_SERVER_URL" >>  /etc/hosts.allow
+mkdir /run/sendsigs.omit.d
```

Comment:
It might be safer to create directories with '-p' flag. Just in case if /run doesn't exist a new one is created.